server/app/services/logging.rb

```diff
@@ -3,7 +3,7 @@
 module Logging
 
   def self.initialize_logger(log_target = STDOUT, log_level = Logger::INFO)
-    @logger = Logger.new(log_target)
+    @logger = Logger.new(ENV['RACK_ENV'] == 'test' ? StringIO.new : log_target)
```

Comment:
These should already be supressed by the `server/spec/spec_helper.rb` when running tests, while still allowing the use of envs to see the logs if required:

```ruby
Logging.initialize_logger((ENV['LOG_TARGET'] || '/dev/null'), (ENV['LOG_LEVEL'] || Logger::UNKNOWN).to_i)
```